.eslintrc.json

```
@@ -0,0 +1,41 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "jest/globals": true,
+    "jest": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "airbnb-base",
+    "airbnb-typescript/base",
+    "plugin:sonarjs/recommended",
+    "plugin:unicorn/recommended",
+    "prettier"
+  ],
+  "overrides": [],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "plugins": ["@typescript-eslint", "jest"],
+  "rules": {
+    "unicorn/filename-case": "off",
+    "import/prefer-default-export": "off",
+    "unicorn/no-null": "off",
+    "unicorn/prevent-abbreviations": "off",
+    "no-console": "warn",
+    "no-restricted-syntax": "off",
+    "import/no-extraneous-dependencies": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/lines-between-class-members": "off",
+    "@typescript-eslint/no-namespace": "off",
+    "no-underscore-dangle": "off",
+    "import/no-cycle": "off",
```

We need those 2 rules.
```suggestion
    "import/no-extraneous-dependencies": "off",
    "@typescript-eslint/lines-between-class-members": "off",
    "@typescript-eslint/no-namespace": "off",
    "no-underscore-dangle": "off",
```